templates/tab/csharp/non-sso/Components/Welcome.razor

```diff
@@ -0,0 +1,74 @@
+﻿@using Microsoft.Extensions.Configuration
+@using Microsoft.AspNetCore.Hosting
+@using Microsoft.Extensions.Hosting
+@inject TeamsFx teamsfx
+@inject TeamsUserCredential teamsUserCredential
+@inject MicrosoftTeams MicrosoftTeams
+@inject IWebHostEnvironment HostEnvironment
+@inject IConfiguration Configuration
+@inject NavigationManager MyNavigationManager
+
+@if(isLoading)
+{
+    <div style="display: flex; justify-content: center; align-items: center; min-height: 100vh;">
+		<FluentProgressRing/>
+	</div>
+}
+else
+{
+    <div class="welcome page">
+        <div class="narrow page-padding">
+            <img src="hello.png" />
+            <h1 class="center">Congratulations!</h1>
+            <p class="center">Your app is running in your @(GetEnvironmentName())</p>
+            <FluentTabs style="width: 80%; justify-content: space-between; margin: 4rem auto;">
+                <FluentTab style="font-size: 14px; min-height: 32px;">1. Build your app locally</FluentTab>
+                <FluentTab>2. Provision and Deploy to the Cloud</FluentTab>
+                <FluentTab>3. Publish to Teams</FluentTab>
+                <FluentTabPanel>
+                    <div style="display:block; width:100%;">
+                        <EditCode />
+                        <AddSSO />
+                    </div>
+                </FluentTabPanel>
+                <FluentTabPanel>
+                    <Deploy/>
+                </FluentTabPanel>
+                <FluentTabPanel>
+                    <Publish/>
+                </FluentTabPanel>
+            </FluentTabs>
+        </div>
+    </div>
+}
+
+@code {
+    string userName;
```

Comment:
Since the Graph component is removed, this userName will no longer used.

Reply:
Update, thx!